app/services/code_climate/api/remote_query.rb

```diff
@@ -0,0 +1,12 @@
+module CodeClimate
+  module Api
+    class RemoteQuery
```

Comment:
What is this class for?

Reply:
It encapsulates the `url` and the parameters of an endpoint such as `.../repos?github_slug='rootstrap/rs-code-review-metrics'